src/modules/settings/pages/About.jsx

```diff
@@ -55,6 +55,7 @@ function About({onHide}) {
       const settingsToImport = isJSON(data);
       if (!settingsToImport) return;
       Object.keys(settingsToImport).forEach((s) => storage.set(s.split('bttv_')[1], settingsToImport[s]));
+      if (!Object.keys(settingsToImport).includes('bttv_settings')) settings.importLegacySettings();
```

Comment:
why do we import all the settings, and then import legacy? why are we calling `Object.keys` again?